src/options.js

```diff
@@ -59,7 +59,7 @@ const translateOptions = (options = {}) => {
   };
   return options;
```

Comment:
Adding previous comment here:

This nested destructuring and assignment is fascinating. 

Could this be simplified to:

```js
let {result = defaults} = options;
```

And then ultimately return the `result` instead of mutating the supplied `options`?